static/js/components/notifications/NotificationContainer_test.js

```diff
@@ -143,4 +145,28 @@ describe("NotificationContainer component", () => {
       1
     )
   })
+
+  //
+  ;[true, false].forEach(filterAlertType => {
+    it(`${shouldIf(filterAlertType)} based on alert types`, async () => {
```

Comment:
The output of this will be something like `should based on alert types`. Probably missing a word or two